pyfr/plugins/tavg.py

```diff
@@ -117,8 +121,8 @@ def _eval_acc_exprs(self, intg):
 
             # Prepare any required gradients
             if self._gradpinfo:
-                # Compute the gradients
-                grad_soln = np.rollaxis(intg.grad_soln[idx], 2)[..., rgn]
+                # Fetch the gradients
```

Comment:
Subset

Reply:
You mean change the comment to # Fetch subset of gradients? Or something more appropriate?